client/my-sites/people/people-list-section-header/test/index.jsx

```diff
@@ -0,0 +1,45 @@
+/**
+ * External dependencies
+ */
+
+import React from 'react';
+import { shallow } from 'enzyme';
+import { expect } from 'chai';
+import sinon from 'sinon';
+
+/**
+ * Internal dependencies
+ */
+import useFakeDom from 'test/helpers/use-fake-dom';
+import useMockery from 'test/helpers/use-mockery';
+
+describe( 'PeopleListSectionHeader', function() {
+	let configMock,
+		PeopleListSectionHeader,
+		SectionHeader;
+
+	useFakeDom();
+
+	useMockery( mockery => {
+		configMock = sinon.stub();
+		configMock.isEnabled = sinon.stub();
+		mockery.registerMock( 'config', configMock );
+		mockery.registerMock( 'lib/analytics', {} );
```

Comment:
are we still left with components using `lib/analytics` or should this have been removed in the change to Redux analytics?